app/src/main/java/com/android/bookswap/resources/C.kt

```diff
@@ -140,6 +140,9 @@ object C {
       const val email = "email"
       const val phone = "phoneNumber"
       const val address = "address"
+      const val image_container = BOOK + UI_T.IMAGE
+      const val profile_image_picture = "profile_image_picture" + UI_T.IMAGE
+      const val profile_image_icon = "profile_image_icon" + UI_T.IMAGE
```

Comment:
`S.OTHERS_USER_PROFILE + "_picture" + UI_T.ICON` is a better tag to differenciate the Async Image from the Icon

Reply:
Okay.
Thank you I will update this.